<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_user_dialog</class>
 <widget class="QDialog" name="search_user_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>197</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>197</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ricerca utente</string>
  </property>
  <widget class="QLabel" name="lbl_tutorial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserire lo username dell'utente che si desidera ricercare:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_query">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>109</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>145</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ricerca utente</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
